LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
A simple question: should this file have an extension? i.e. should it be `LICENSE.md` or `LICENSE.txt` rather than just `LICENSE`?

Reply:
I suggest that it be named LICENSE.md to be similar to the CODE_OF_CONDUCT.md file name (although I do not know what ".md" is supposed to signify).

Reply:
There was just a discussion about this in the `spacetelescope:style-guide` repo. Apparently `LICENSE` is preferred because the file contains no code of any kind (including markdown, which is what .md stands for). It sounds like the other acceptable option is `LICENSE.txt`.

Reply:
Changed to LICENSE.txt